packages/console/src/pages/Connectors/components/ConfigForm/ConfigForm.module.scss

```diff
@@ -0,0 +1,7 @@
+@use '@/scss/underscore' as _;
```

Comment:
the PR title and description looks too informal?

Reply:
Updated.